bundles/org.openhab.binding.volumio/src/main/resources/OH-INF/i18n/volumio.properties

```diff
@@ -0,0 +1,15 @@
+# add-on
+addon.volumio.name = Volumio Binding
+addon.volumio.description = This is a binding for a volumio device.
+
+# thing types
+thing-type.volumio.player.label = Volumio Binding Thing
+thing-type.volumio.player.description = Volumio Player
+
+# thing type config description
+thing-type.config.volumio.player.hostname.label = Hostname
+thing-type.config.volumio.player.hostname.description = The hostname of the Volumio Player.
+
+# channel types
+channel-type.volumio.systemCommand.label = Send system command
+channel-type.volumio.systemCommand.description = Sends a system command to the Volumio Player. This allows a shutdown/reboot.
```

Comment:
I would have expected more content in this file based on the `thing-types.xml`

Reply:
resolved